crates/uv/src/commands/venv.rs

```diff
@@ -121,18 +119,11 @@ async fn venv_impl(
     cache: &Cache,
     printer: Printer,
 ) -> miette::Result<ExitStatus> {
-    // Locate the Python interpreter.
-    let interpreter = if let Some(python) = python_request.as_ref() {
-        let system = uv_interpreter::SystemPython::Required;
-        let request = InterpreterRequest::parse(python);
-        let sources = SourceSelector::from_settings(system, preview);
-        find_interpreter(&request, system, &sources, cache)
-    } else {
-        find_default_interpreter(preview, cache)
-    }
-    .into_diagnostic()?
-    .into_diagnostic()?
-    .into_interpreter();
+    // Locate the Python interpreter to use in the environment
+    let interpreter =
+        PythonEnvironment::find(python_request, SystemPython::Required, preview, cache)
```

Comment:
Does `uv venv` from within a venv still work?

Reply:
I'll verify this — probably not. I think the `find_default_interpreter` branch is actually important here.

Thanks for flagging.

Reply:
I wonder if we can add test coverage for that case too...

Reply:
So what I have here is actually equivalent to the current code. I think `uv venv` from an active virtual environment doesn't read the interpreter from the virtual environment. Should it? Maybe this regressed in #3266 and we missed it because we didn't have test coverage?

Reply:
Discussed in Discord, the behavior that Charlie mentioned was changed in #3266 since we skip interpreters in the PATH that are not virtual environments.

There is no behavior change in this pull request and I think the existing behavior is okay for now since we haven't gotten any reports about the change that was released in 0.2.0.